app/components/primer/popover_component.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module Primer
+  class PopoverComponent < Primer::Component
+    include ViewComponent::Slotable
+
+    with_slot :message, :heading, :body, :button
```

Comment:
Check the component now, I dropped `body` and just use `message.content` in its place. The structure I have now looks like it will support what I was trying to do with manual Primer `.Popover` HTML before.

Reply:
Hmm this is a tough one - I think we previously had different sub components for the message/heading/button/etc in Primer React Components but didn't find it to be flexible enough for all the use cases. That being said, I do like that this makes the Popover a bit more opinionated 🤔 It won't allow folks to use any button or not have a button, or put something weird like a select menu inside of the Popover. @cheshire137 do you have a code snippet of how you used this in .com that you could share with me (maybe on Slack since this repo is public)?

Reply:
Took a look at @cheshire137 usage in .com and I think this API is reasonable. It will be a good way to dip our toes into being a little more opinionated, let's give it a shot! Nice work on this @cheshire137! 👏👏👏👏